astroquery/hsa/tests/test_hsa_remote.py

```diff
@@ -0,0 +1,117 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+import pytest
+
+import os
+
+from ..core import HSAClass
+from .test_hsa import TestHSA
```

Comment:
please don't import this class here, it would be then rerun all its tests methods 2 more times.

Reply:
👍